templates/templates/docs/openstack/search.html

```diff
@@ -0,0 +1,85 @@
+{% extends "templates/base.html" %}
+
+{% block title %}Search results{% if query %} for "{{ query }}"{% endif %} | Charmed OpenStack docs{% endblock %}
+
+{% block outer_content %}
+
+  <div class="p-strip is-shallow">
```

Comment:
Same here, this template is almost identical to the other search results pages for the other docs, so shouldn't it be in an include like `shared/docs/_search-results.html`?